examples/app.py

```diff
@@ -29,12 +29,20 @@
 
 .. code-block:: console
 
-    $ # install the sample application
-    $ cd examples/samplepkg
-    $ pip install -e .
-    $ # run the example application
-    $ cd ..
+    $ pip install -e .[all]
+    $ ./app-setup.sh
```

Comment:
The solution would be to include the file in `docs/usage.rst` or similar ... but IMHO it's not a blocker.